docs/appendix/troubleshooting.md

```diff
@@ -158,6 +158,68 @@ It will resemble the following:
 }
 ```
 
+### Error: failed to find camera
+
+**Additional Errors:** `cannot open webcam`, and `found no webcams`.
+
+**Description:** When working with a [camera](/components/camera/) component on the Linux platform, your Linux OS must be able to access the camera properly, and the camera must be configured to use a pixel format that Viam supports.
+
+**Solution:** On your Linux system, verify each of the following:
+
+- Ensure that your Linux OS is able to access your camera:
+
+  1.  First, find your video device:
+
+      ```sh {class="command-line" data-prompt="$"}
+      ls -l /dev/v4l/*
+      ```
+
+      In the list of camera devices returned, find the entry that matches the `video_path` from your camera component's configuration in the Viam app.
+      For example, if your camera is configured with a `video_path` of `usb-GENERAL_GENERAL_WEBCAM-video-index0`, you would find it in the output of the above command like so:
+
+      ```sh
+      usb-GENERAL_GENERAL_WEBCAM-video-index0 -> ../../video0
+      ```
+
+      The video device number is the number after `video` appearing on the symlink target for that device, in this case `0`.
+
+  1.  Then, [stop `viam-server`](/installation/manage/#run-viam-server), and verify that your Linux OS is able to access that video device properly:
+
+      ```sh {class="command-line" data-prompt="$"}
+      v4l2-ctl -d0 --stream-mmap
+      ```
+
+      Replace the `0` in the above command with the number you determined for your video device above.
+      You should receive output resembling like the following, which indicates that your Linux OS is successfully able to use your video device:
+
+      ```sh
+      <<<<<<<<<<<<<<<<<<<<<<<< 22.81 fps
+      <<<<<<<<<<<<<<<<<<<<<<<< 23.50 fps
+      ```
+
+      If the command does not return FPS readings as shown above, consult the documentation for your camera and Linux distribution to troubleshoot.
+      If you receive the error `Device or resource busy` instead, be sure you have [stopped `viam-server`](/installation/manage/#run-viam-server) first, then re-run the command above.
+
+- Ensure that your camera uses a supported pixel format:
+
+  1.  First, determine your video device number, like `0`, following the instructions above.
+  1.  Then, run the following command:
+
+      ```sh {class="command-line" data-prompt="$"}
+      v4l2-ctl --list-formats-ext --device /dev/video0
```

Comment:
It is completely optional, but to make things simpler, you can refer to `/dev/video0` as a video path throughout this document. You can also replace the `-d0` arguments with `--device /dev/video0` throughout as well for consistency. Calling it "video path" is consistent with our setup steps for the camera component. That'll turn phrases like
> ... determine your video device number, like `0`,...

to
 
> ... determine your video path, ...

Reply:
Thanks so much, this is a much better approach!